run.sh

```diff
@@ -38,18 +76,24 @@ if [[ "x$1" != "x--restart" ]]; then
   fi
 fi
 
-pushd .
-  # Ensure rustc is updated
-  echo "* Updating rust if necessary..."
+pushd . &> /dev/null
   cd runtime
-  multirust override nightly-2015-08-10
 
-  # Compile runtime server
-  echo "* Compiling server... (This takes a while)"
+  # Ensure rustc is updated.
+  multirust list-toolchains | grep "$rustVersion" &> /dev/null
```

Comment:
I confirmed that we need to override irrespective of whether it's installed or not. Since multiple rust versions can be installed in multi-rust, we need to tell it to use the appropriate version. I do like the clearer diagnostic output though. Also, for consistency, in the case that rust does need updated it would be nice to use printf and show completion like you are elsewhere.